frontend/src/lib/util.ts

```diff
@@ -230,6 +235,59 @@ export function useErrorState(dependentSetter?: (...args: any) => void) {
   return [error, setError as any];
 }
 
+// Source Code: https://github.com/kubernetes/kubernetes/blob/2f275b72b2248bb7c522327115acb17223b8387b/staging/src/k8s.io/apimachinery/pkg/util/duration/duration.go#L48
```

Comment:
You add this function back. Why not use `humanize-duration` package?

Reply:
This was a mistake. I'll remove it.